src/viam/media/video.py

```diff
@@ -183,50 +80,24 @@ def mime_type(self, value: CameraMimeType):
         if value == self.mime_type:
             return
         self._mime_type = value
-        self.close()
-        self._image_decoded = False
-        self._image = None
 
     @property
-    def image(self) -> Optional[Image.Image]:
-        """The PIL.Image representation of the image. If the mime type is not supported, this will be None."""
-        if not CameraMimeType.is_supported(self.mime_type):
-            self._image = None
-            self._image_decoded = True
-            return self._image
-
-        try:
-            self._image = Image.open(BytesIO(self.data), formats=LIBRARY_SUPPORTED_FORMATS)
-        except UnidentifiedImageError:
-            self._image = None
-        self._image_decoded = True
-        return self._image
-
-    def close(self):
-        """Close the image and release resources."""
-        if self._image is not None:
-            self._image.close()
-
-    def bytes_to_depth_array(self) -> List[List[int]]:
-        """Decode the data of an image that has the custom depth MIME type ``image/vnd.viam.dep`` into
-        a standard representation.
-
-        Raises:
-            NotSupportedError: Raised if given an image that is not of MIME type `image/vnd.viam.dep`.
+    def width(self) -> Union[int, None]:
+        """The width of the image"""
+        return self._width
```

Comment:
Continuing from my earlier comment around keeping the `_image` and `_image_decoded` properties, the `width` and `height` getters should now be something like:

```python3
# First, check if we already have a value
if self._width is not None:
    return self._width
# If we don't have a value, check to see if we have already tried to decode the image
# If we have not, then try to decode the image now
if not self._image_decoded:
    try:
        self._image = Image.open(BytesIO(self.data), formats=LIBRARY_SUPPORTED_FORMATS)
    except UnidentifiedImageError:
        self._image = None
    self._image_decoded = True
# If we have decoded the image and the image is not none, then set the width so we don't have to do this again
if self._image_decoded and self._image is not None:
    self._width = self._image.width
return self._width
```